Doc/library/os.rst

```diff
@@ -1127,13 +1127,15 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
    .. versionadded:: 3.7
 
 .. data:: RWF_DSYNC (since Linux 4.7)
```

Comment:
I would add the line
```
Availability: Linux (version 4.7).
```
after the description.

Reply:
Yes, eliminate it. Currently `(since Linux 4.7)` is rendered as a signature of the RWF_DSYNC callable.

Reply:
@serhiy-storchaka Done in b8eea4e